src/js/common/components/Modal.jsx

```diff
@@ -32,8 +35,10 @@ class Modal extends React.Component {
 
   componentWillReceiveProps(newProps) {
     if (newProps.visible && !this.props.visible) {
+      document.addEventListener('keyup', this.handleDocumentKeyUp, false);
       this.setState({ lastFocus: document.activeElement, focusListener: focusListener(newProps.focusSelector) });
     } else if (!newProps.visible && this.props.visible) {
+      document.removeEventListener('click', this.handleDocumentClick, false);
```

Comment:
What is `handleDocumentClick`? Doesn't seem to be defined in this class.

Reply:
Wow, good call. Not sure how that wound up there.